services/ui-src/src/measures/2021/CommonQuestions/OptionalMeasureStrat/context.ts

```diff
@@ -1,12 +1,13 @@
 import { createContext, useContext } from "react";
 import * as Types from "../types";
 
-export type CompFlagType = "DEFAULT" | "IU" | "PCR";
+export type CompFlagType = "DEFAULT" | "IU" | "PCR" | "AIF";
```

Comment:
```suggestion
export type CompFlagType = "DEFAULT" | "AIF" | "IU" | "PCR";
```
Alphabetized because I'm that guy